src/fmke_driver_opt_aql.erl

```diff
@@ -0,0 +1,756 @@
+%% --------------------------------------------------------------
+%% Database driver for AQL, an SQL-like interface for AntidoteDB.
```

Comment:
Feel free to add a link to any documentation about AQL here as well, if applicable.

Reply:
Done.